frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -964,21 +976,18 @@ export default function ToolbarView() {
             value={AnnotationToolEnum.SKELETON}
           >
             {/*
-           When visible changes to false, the tooltip fades out in an animation. However, skeletonToolHint
-           will be null, too, which means the tooltip text would immediately change to an empty string.
-           To avoid this, we fallback to previousSkeletonToolHint.
-          */}
-            <Tooltip
-              title={skeletonToolHint || previousSkeletonToolHint}
-              open={skeletonToolHint != null}
```

Comment:
this didn't really work, anyway, and it's also not that important because we have the status bar which also shows how modifiers change the tool behavior.